docs/source/deploying.rst

```diff
@@ -1,12 +1,43 @@
 .. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-Deploying to Daml Ledgers
-=========================
+Overview of Daml ledgers
```

Comment:
I thought the plan was that we drop this completely for 2.0 and instead put this on daml.com instead of docs.daml.com?

Reply:
In a meeting this morning we decided to update the content until we can get a spot in the new digitalasset.com site that's currently being built.